packages/text-field/index.tsx

```diff
@@ -329,23 +315,23 @@ class TextField<T extends HTMLElement = HTMLInputElement> extends React.Componen
         <div
           {...this.otherProps}
           className={this.classes}
-          onClick={() => foundation && foundation.handleTextFieldInteraction()}
-          onKeyDown={() => foundation && foundation.handleTextFieldInteraction()}
+          onClick={() => foundation!.handleTextFieldInteraction()}
```

Comment:
can you change this to `onClick={foundation. handleTextFieldInteraction}`? Same with onKeyDown handler

Reply:
Can you tell me what is this syntax mean?
`onClick={foundation. handleTextFieldInteraction}`
Causes an error!

Reply:
foundation.handleTextFieldInteraction is a function. So the onClick event will trigger function.

Reply:
Ah I see, I confused because of space before "handleTextFieldInteraction".
I reflect it!